app/views/keywords/_list_keywords.html.erb

```diff
@@ -1,12 +1,28 @@
 <section class="list-keyword" data-turbo="true">
   <%= turbo_frame_tag 'frame_list_keywords' do %>
+    <% if filter_error.blank? %>
+      <div class="d-flex justify-content-end match-counts">
```

Comment:
Based on our CSS conventions, lets use singular class name(same for other plural css classes)
```suggestion
      <div class="d-flex justify-content-end match-count">
```